src/main/java/org/folio/linked/data/mapper/dto/monograph/instance/InstanceMapperUnit.java

```diff
@@ -75,8 +77,8 @@ public <P> P toDto(Resource source, P parentDto, Resource parentResource) {
     if (parentDto instanceof ResourceResponseDto resourceDto) {
       var instance = coreMapper.toDtoWithEdges(source, InstanceResponse.class, false);
       instance.setId(String.valueOf(source.getId()));
-      instance.setInventoryId(source.getInventoryId());
-      instance.setSrsId(source.getSrsId());
+      instance.setInventoryId(source.getInstanceInventoryId());
```

Comment:
TODO - Remove the separate fields "inventoryId" and "srsId" from response. Instead, include "instanceMetadata" DTO in response.